examples/c/wind_turbine_nmpc.c

```diff
@@ -667,8 +760,14 @@ int main()
 		}
 		else if (plan->sim_solver_plan[i].sim_solver == NEW_LIFTED_IRK)
 		{
-			sim_opts->ns = 4;
-			sim_opts->num_steps = 1;
+			sim_opts->ns = 5;
```

Comment:
why are those changed? Does the new_lifted_irk even work for num_steps > 1?

Reply:
As far as read into that, it should work. But I'll undo this change.

Reply:
The values you chose are anyway to high for practical applications. Ping @zanellia for what is currently supported in the new lifted IRK.